src/box.jl

```diff
@@ -18,6 +18,9 @@ Box(c::SVector{N}, d::SVector{N},
 Box(c::AbstractVector, d::AbstractVector, axes=eye(length(c)), data=nothing) =
     (N = length(c); Box(SVector{N}(c), SVector{N}(d), SMatrix{N,N}(axes), data))
 
+Box(b::NTuple{2,AbstractVector}, axes=eye(length(b[1])), data=nothing) =
```

Comment:
I don't think it is a big imposition to ask users to call a different function name if they want the arguments to be interpreted differently.   Julia has lots of functions that construct arrays precisely for this reason.